server/api/appointment_handler.py

```diff
@@ -11,7 +11,7 @@
 
 
 @appointment_handler.route('/appointment/create', methods=['POST'])
-#@check_token
+@check_token
```

Comment:
I was trying to delete this because the user who is booking on calendar doesn't require to login

Reply:
oh yea true, will remove